docs/user-guide/a2-01-spirv-target-specific.md

```diff
@@ -291,13 +234,13 @@ When there are more than one entry point, the default behavior will prevent a sh
 To generate a valid SPIR-V with multiple entry points, use `-fvk-use-entrypoint-name` compiler option to disable the renaming behavior and preserve the entry point names.
 
 
-Memory pointer is experimental
+Global memory pointers
 ------------------------------
```

Comment:
Does markup care about the length of underlines?

Reply:
No it doesn't.